tests/register.spec.coffee

```diff
@@ -44,6 +29,26 @@ describe 'Device Register:', ->
 
 	describe '.register()', ->
 
+		beforeEach ->
+			mockServer.start()
+
+		beforeEach ->
```

Comment:
NIT: this could be a `before` and an `after` (not `beforeEach`) with the mock and the server start.

Reply:
We are using it inside a beforeEach in balena-request and thought to keep it this way in here as well, but I will change it :+1: